src/build.sh

```diff
@@ -311,7 +311,7 @@ for branch in ${BRANCH_NAME//,/ }; do
         # Start the build
         echo ">> [$(date)] Starting build for $codename, $branch branch" | tee -a "$DEBUG_LOG"
         build_successful=false
-        if brunch "$codename" &>> "$DEBUG_LOG"; then
+        if brunch "$codename $BUILD_TYPE" &>> "$DEBUG_LOG"; then
```

Comment:
Since it's in quotes, this becomes a single argument. Shouldn't this be `brunch "$codename" "$BUILD_TYPE"`?

Reply:
@bananer Now that you mention it, it would seem that way, I have been running the above code without issue, but I will make the change and test it just to be sure.

Reply:
![image](https://user-images.githubusercontent.com/8284579/151067328-6529f7c6-4b26-4c4a-8551-474c41dfec25.png)

This is after splitting the parameters.